src/test/resources/cristin_one_person_invalid_id.json

```diff
@@ -0,0 +1,7 @@
+{
+  "status": 400,
```

Comment:
What about 404 Not Found? Maybe its not needed

Reply:
Possibly a better approach would be to generate these files in our tests rather than having to maintain all of these variants.

Reply:
Added a 404 case and test.